make.py

```diff
@@ -184,8 +191,15 @@ def stage(text):
     assert os.path.exists('bindings.bc')
 
     stage('Building OpenCV.js')
-    opencv = os.path.join('..', '..', 'build', 'cv.js')
-    data = os.path.join('..', '..', 'build', 'cv.data')
+
+    if clArguments.wasm:
+        emcc_args += "-s WASM=1".split( " " )
+        basename = "cv-wasm"
+    else:
+        basename = "cv"
+
+    destFiles = [ os.path.join('..', '..', 'build', basename + ext ) for ext in [ ".js", ".data", ".wasm" ] ]
+    opencv = destFiles[0]
```

Comment:
Well, putting all the files in the same folder with different names seemed simpler to me for the time being, as the .js file generated by emscripten expects to find its related .data (and .wasm, in case of WebAssembly) in the same folder as the .html file.
Having all the files in the same folder with different names depending on the technology used (WebAssembly or asm.js) simplifies the use: the user can just copy everything in build/ to the folder of his/her .html file, and then either load cv.js or cv-wasm.js depending on what technology s/he wants to use.